pkg/pdcli/oncall/handlers.go

```diff
@@ -30,16 +36,24 @@ func TeamSREOnCall(c client.PagerDutyClient) ([]OncallUser, error) {
 		constants.OncallIDWeekend,
 		constants.InvestigatorID,
 	}
-
+	since := time.Now().Add(time.Hour * -11)
+	until := time.Now().Add(time.Hour * 13)
+	callOpts.Since = since.String()
+	callOpts.Until = until.String()
+	callOpts.Limit = 100
 	// Fetch the oncall data from pagerduty API
 	oncallListing, err := c.ListOnCalls(callOpts)
 
 	if err != nil {
 		return nil, err
 	}
 
+	startTime, _ := utils.FormatTimestamp(oncallListing.OnCalls[0].Start)
+	var temp []OncallUser
+	var mgmtUsers []OncallUser
```

Comment:
`mgmtUsers`?

Reply:
For a Oncall Schedule of any region, the role of 1-SREP: Management Escalation remains the same and has a different start and end time. Thus to add this user to all oncall layer item, I keep a track of it in the mgmtUsers slice